tests/suite-unit/PluginFlyvemdmAgent.php

```diff
@@ -161,39 +165,40 @@ public function testDefineTabs() {
    /**
     * @tags testShowForm
     */
-   /*public function testShowForm() {
+   public function testShowForm() {
+      $agent = $this->createAgent([]);
       $instance = $this->newTestedInstance();
       ob_start();
-      // TODO: have a fake agent registered in DB before this.
-      $instance->showForm(1);
+      $instance->showForm($agent->getID());
       $result = ob_get_contents();
       ob_end_clean();
       $this->string($result)
-         ->contains("method='post' action='-/plugins/flyvemdm/front/agent.form.php'")
+         ->matches("#method='post' action='.+?\/plugins\/flyvemdm\/front\/agent\.form\.php'#")
```

Comment:
If you use a regex here, why not, but havilg a lot of backslashes makes harder to red it / debug it.

Consider using preg_quote http://php.net/manual/en/function.preg-quote.php to deletage the escaping and make long literal strings more readable, or an other method of your choice.